tests/rolling_deploy_test.py

```diff
@@ -269,6 +273,17 @@ def test_double_autoscale_instance_count(self):
   def test_decrease_autoscale_instance_count(self):
     self.assertEqual(self.rolling_deploy.decrease_autoscale_instance_count(4), 2)
 
+  def test_cloudwatch_filter_function_with_no_alarm(self):
+    bad_alarm = MetricAlarm(name="xyz_CloudWatchAlarm", metric="CPUUtilization", namespace="AWS/EC2")
+    cloud_watch_alarms = filter(lambda alarm : self.rolling_deploy.project in alarm.name and self.rolling_deploy.env in alarm.name, [ bad_alarm])
+    self.assertEqual(0, len(cloud_watch_alarms))
+
+  def test_cloudwatch_filter_function_with_1_alarm(self):
+    bad_alarm = MetricAlarm(name="xyz_CloudWatchAlarm", metric="CPUUtilization", namespace="AWS/EC2")
+    good_alarm = MetricAlarm(name=self.GMS_CLOUDWATCH_GROUP_STG, metric="CPUUtilization", namespace="AWS/EC2")
+    cloud_watch_alarms = filter(lambda alarm : self.rolling_deploy.project in alarm.name and self.rolling_deploy.env in alarm.name, [bad_alarm, good_alarm])
+    self.assertEqual(1, len(cloud_watch_alarms))
```

Comment:
@prabhakar1983  please use the actual methods in the class you are trying to unit test (instead of recreating logic)